msteams-platform/sbs-calling-and-meeting.yml

```diff
@@ -107,10 +125,19 @@ items:
      > * If an app isn't granted IT admin consent, users must provide consent the first time they use an app.
      > * Users need to consent to the API permissions only if the Microsoft Entra app is registered in a different tenant.
 
+     > [!div class="nextstepaction"]
+     > [I ran into an issue](https://github.com/MicrosoftDocs/msteams-docs/issues/new?template=Doc-Feedback.yaml&title=%5BI+ran+into+an+issue%5D+Add+API+permission+for+downstream+calls&pageUrl=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fmicrosoftteams%2Fplatform%2Fsbs-calling-and-meeting%3Ftabs%3Ddev%26tutorial-step%3D3&contentSourceUrl=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fblob%2Fmain%2Fmsteams-platform%2Fsbs-calling-and-meeting.yml&documentVersionIndependentId=6760feec-169b-a49b-ddde-1fd363943a1a&author=%40surbhigupta&metadata=%2A%2BID%253A%2Be473e1f3-69f5-bcfa-bcab-54b098b59c80%2B%250A%2A%2BService%253A%2B%2A%2Amsteams%2A%2A)
```

Comment:
Please update title as;

Add API permission**s** for downstream calls

Reply:
Updated as suggested